frontend/src/Editor/Components/DropdownV2/CustomOption.jsx

```diff
@@ -14,7 +14,7 @@ const CustomOption = (props) => {
           </span>
         )}
         <span style={{ color: props.isDisabled ? '#889096' : 'unset', wordBreak: 'break-all' }}>
-          {highlightText(props.label, props.selectProps.inputValue)}
+          {highlightText(props.label.toString(), props.selectProps.inputValue)}
```

Comment:
Can we add a null check for label? Else if label is undefined or null from a resolved value it might crash